src/components/progress-bar/progress-bar.spec.ts

```diff
@@ -0,0 +1,143 @@
+import {beforeEach, ddescribe, expect, inject, it, TestComponentBuilder} from 'angular2/testing';
+import {Component, DebugElement} from 'angular2/core';
+import {By} from 'angular2/platform/browser'
+import {MdProgressBar} from './progress-bar';
+
+
+export function main() {
+  describe('MdProgressCircular', () => {
+    let builder:TestComponentBuilder;
```

Comment:
As well as everywhere. If you're using the default TypeScript formatting in WebStorm it will do that, you can change it in the preferences (Editor > Code Style > TypeScript > Spaces (tab) > [x] After type reference colon)